src/lib/logo.js

```diff
@@ -0,0 +1,14 @@
+export function logoSetup() {
+  const logoUrl = "https://rickandmortyapi.com/api/character/";
+
+  const logo = document.querySelector(".header__img");
+
+  fetch(logoUrl)
+    .then((response) => {
+      return response.json();
+    })
+    .then((dataObject) => {
+      logo.src = dataObject.results[18].image;
+      logo.style.borderRadius = "50%";
+    });
+}
```

Comment:
Since you not changing the logo dynamically would it be better to "just" download the image and put in hard coded in the code? You can save one request. (I know you want to try `fetch()` out ;-)) But sometimes api calls "costs".

Reply:
Damnit, I thought I was doing something clever here ;D